# phylogenetic-analysis
PHYLOGENETIC ANALYSIS OF DNA SEQUENCES USING FOURIER TRANSFORM

In phylogenetic analysis, branching diagrams are made to represent the evolutionary history or relationship between different species, organisms, or characteristics of an organism (genes, proteins, organs, etc.) that are developed from a common ancestor. The diagram is known as a phylogenetic tree

There are two types of phylogenetic trees:-

1)Rooted phylogenetic tree (cladogram)

2)Unrooted phylogenetic tree (Phenogram)

DATA SETS:
Worked on two different datasets:​
1) 21 Mycobacterial genomes (Tuberculosis and non-Tuberculosis strains)​
2) Myeloid cell leukaemia protein (AAG00896) original and mutated sequences.

Data set can be downloaded from here - https://drive.google.com/file/d/1NcKbxCFIS3M5HknARie80XviRBbIxe_P/view?usp=drive_link

Methods used for construction of phylogenetic tree - Distance matrix method

Mapping scheme used: VOSS

UPGMA: Unweighted Pair Group Method with Arithmetic Mean: A simple clustering method that assumes a constant rate of evolution (molecular clock hypothesis). It needs a distance matrix of the analysed taxa that can be calculated from a multiple alignment.
